worlds/yoshisisland/Regions.py

```diff
@@ -0,0 +1,225 @@
+from typing import List, Dict, Tuple
+from BaseClasses import MultiWorld, Region, Entrance, Location
+from .Locations import LocationData
+from .level_logic import YoshiLogic
+from .setup_bosses import BossReqs
+
+class YILocation(Location):
+    game: str = "Yoshi's Island"
+    LevelID: int
+
+def __init__(self, player: int, name: str = " ", address: int = None, parent=None, LevelID: int = None):
+    super().__init__(player, name, address, parent)
+    self.LevelID = LevelID
+
+
+def init_areas(world, locations: Tuple[LocationData, ...]):
+    multiworld = world.multiworld
+    player = world.player
+    location_cache = world.location_cache

```

Comment:
why does this exist?

Reply:
Because this was specifically commented for me to add.